### DISCUSSION TITLE
Symlink to 'latest' release?

### DISCUSSION THREAD

**Discussion by danmanix** [OvertureMaps/data#209] [Discussion]
### Category Feedback
Any plans to add a symlink to the 'latest' version?

On S3 at least, its pretty clumsy finding the latest release programmatically and prone to breaking if a non release file happens to be the latest (without even more code to check filename formats).

### Dependency with other categories, if any.

Applies to all - there was no 'general data feedback' topic

**Comment by jwass** [comment]
Unfortunately S3 does not provide a symlink capability. But I agree - I'd love to have it to redirect people to the latest release.

We've had discussions about some kind of meta/manifest file to list out releases, their locations, etc that would have broad applicability across all our tools but haven't quite finalized on anything yet. One simple PR is here: https://github.com/OvertureMaps/data/pull/186

**Reply by danmanix** [reply]
Thanks for the reply. Yeah I saw the PR, and that of course would do the job. There are redirects available in S3, but you've probably got bigger fish to fry.
